packages/react/src/drafts/IssueLabelToken/getColorsFromHex.ts

```diff
@@ -0,0 +1,83 @@
+import {getContrast} from 'color2k'
+import {Hsluv} from 'hsluv'
+
+export const getColorsFromHex = (colorHex: string, colorScheme = 'light', isSelected = false, bgColor: string) => {
```

Comment:
Could you add more details or a summary to the code here? It seems like it's doing a lot of important stuff but I don't necessarily know what each of the numbers are for so any additional info in the source would be very appreciated 😅